File: python/prism_sdk/throughput_lease_fencing_integrity_inference.py
"""Factory P32 throughput lease/fencing integrity feature."""
from .lease_fencing_integrity_support import *
FEATURE_ID="AFA-factory-P32-F03"; CONTRACT_VERSION="factory-throughput_lease_fencing_integrity_inference/1.0"
def throughput_lease_fencing_integrity_inference_manifest():return manifest(feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="throughput",mode="inference")
def qualify_throughput_lease_fencing_integrity_inference(request):return qualify(request,feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="throughput",mode="inference")

File: crates/factory/src/throughput_lease_fencing_integrity_inference.rs
//! Factory P32 throughput lease/fencing integrity feature.
use super::lease_fencing_integrity_support::{
    manifest, qualify, LeaseFencingIntegrityCard7, LeaseFencingIntegrityError,
    LeaseFencingIntegrityRequest4,
};
pub const FEATURE_ID: &str = "AFA-factory-P32-F03";
pub const CONTRACT_VERSION: &str = "factory-throughput_lease_fencing_integrity_inference/1.0";
pub fn throughput_lease_fencing_integrity_inference_manifest() -> serde_json::Value {
    manifest(FEATURE_ID, CONTRACT_VERSION, "throughput", "inference")
}
pub fn qualify_throughput_lease_fencing_integrity_inference(
    q: &LeaseFencingIntegrityRequest4,
) -> Result<LeaseFencingIntegrityCard7, LeaseFencingIntegrityError> {
    qualify(q, FEATURE_ID, CONTRACT_VERSION, "throughput", "inference")
}
